### ISSUE TITLE
honnef.co/go/tools/unused: TestAll fails with Go 1.14 beta1

### ISSUE THREAD

**Issue by eclipseo** [dominikh/go-tools#687]
Fedora Rawhide with Go 1.14 beta1:

```
honnef.co/go/tools/unused
--- FAIL: TestAll (0.32s)
    unused_test.go:143: /builddir/build/BUILD/go-tools-2019.2.3/_build/src/honnef.co/go/tools/unused/testdata/src/tests/tests.go:3:6: unexpected: fn
FAIL
exit status 1
FAIL	honnef.co/go/tools/unused	0.331s
```

**Comment by dominikh** [comment]
This was fixed in 7398d928b10cbeeef873ee02a66742af75a1d4e9 (coincidentally) when we updated our dependency on golang.org/x/tools. The commit in golang.org/x/tools that seems to have fixed this failure is golang/tools@db0687cefd95dc155728a53f4a5416d1aa1bcf1f.

I am planning to release Staticcheck 2020.1 shortly before Go 1.14 releases. In the meantime, you should be able to apply the following patch:

```
diff --git a/go.mod b/go.mod
index 9ae497b9..7d4e3801 100644
--- a/go.mod
+++ b/go.mod
@@ -8,5 +8,5 @@ require (
        github.com/kisielk/gotool v1.0.0
        github.com/rogpeppe/go-internal v1.3.0
        golang.org/x/mod v0.0.0-20190513183733-4bf6d317e70e
-       golang.org/x/tools v0.0.0-20190826182127-07722704da13
+       golang.org/x/tools v0.0.0-20191022074931-774d2ec196ee
 )
```

**Comment by eclipseo** [comment]
Ha thanks, I haven't updated my golang.org/x/tools yet.
